src/torrent.cpp

```diff
@@ -60,11 +60,8 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <functional>
 
 #ifdef TORRENT_SSL_PEERS
+#include "libtorrent/ssl.hpp"
```

Comment:
ssl.hpp is included twice in this file. Once conditionally (here), and then unconditionally a bit further down. I imagine this include is redundant.